voluntary-exit.sh

```diff
@@ -1,72 +0,0 @@
-#!/bin/bash
```

Comment:
Does this no longer apply?

Reply:
This script turned obsolete as it only exits the teku vc. I will create a new story to add exit templates for the other VCs (LH, lodester, and vouch).

Reply:
Ah ok cool 👌